user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -216,7 +218,7 @@ three options:
 * Test this media and install Qubes OS
 * Troubleshooting
  
-Select the option to test this media and install Qubes OS. 
+Select the option to test this media and install Qubes OS, or (**Advanced hard headed users ONLY**) install without any checking as you already know the hardware is safe. 
```

Comment:
I don't think this is necessary or desirable, as advanced users already know that they can skip the media check. Skipping doesn't save much time anyway. Also, it's unclear what "safe" means in this context, as we're not talking about hardware _security_ here.